src/the-2021-scheduler.md

```diff
@@ -7,7 +7,7 @@ scheduler](./the-anki-2.1-scheduler.md) ("v2").
 
 As of Anki/AnkiMobile 23.10, and AnkiDroid 2.17, the v3 scheduler is the default and only option.
 
-On older versions, the scheduler can be changed in the preferences screen.
+On earlier versions, the scheduler can be changed from the preferences screen.
```

Comment:
The preposition *from* felt more natural here. Although, I agree I could've kept the *in* in there.

As for the word *old*, I think it comes with a negative connotation oftentimes. Using *earlier* is neutral and is more clear. One might think, because they downloaded Anki a few months ago, it's probably not "old".